tests/10-vcdm2.js

```diff
@@ -19,6 +19,13 @@ const baseContextUrl = 'https://www.w3.org/ns/credentials/v2';
 const tag = 'vc2.0';
 const {match} = filterByTag({tags: [tag]});
 
+// used with VP creation and verification throughout
+const createOptions = {challenge};
+const verifyPresentationOptions = {
+  checks: ['proof'],
```

Comment:
```suggestion
```

Reply:
I suggest to remove the checks option.

Reply:
Is there more needed at this point beyond the removal of that option?

Reply:
@PatStLouis @BigBlueHat so DB needs `checks: ['proof']` but other implementers might not need that option and further `options.checks` has not be in the VC API for awhile so the best way to handle this is to add `options.checks` to the DB implementation and do something like this:

https://github.com/w3c-ccg/data-integrity-test-suite-assertion/blob/6d61d0f3940694edb12437bcb0f457a8b7cb56cb/assertions.js#L25-L46

How to handle this situation in the bitstring status list suite is another issue as those tests will probably need us to expose a second verify with the correct checks for our verifier.

Reply:
@PatStLouis definitely looks like we should change this, but I'd prefer to not block this PR for that. I'll make an issue, if you wouldn't mind re-reviewing this one, that'd be great. Thanks!

Reply:
@BigBlueHat so the DB verifier now has the checks in the options here: https://github.com/w3c/vc-test-suite-implementations/blob/9a014669b6f62ed9c1a920ef9e054c510abefe6c/implementations/DigitalBazaar.json#L82-L91

so you can now pass the verifier's implementation specific options to each verifier which might result in failure or more success LMK.

Reply:
Thanks! I'd still rather do it in a separate PR as doing it here will cause conflicts with all the other pending PRs...

Reply:
separate PR sounds like the right approach